css/style.css

```diff
@@ -29,7 +29,8 @@ video {
 	position: relative;
 	margin: 0 auto;
   max-width: 960px;
-  width: 100%;
+  height: 300px;
```

Comment:
`.container` is used through all the site, please set the height in a new class that is specific for the map, also restore the `width: 100%;` rule.